sandbox/src/mocks/cancelReferral/responses/ExampleCancelledReferral.json

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
The example responses do not have the cancel reason or comment in either the referral or appointment

Reply:
ERSSUP-60025 will fix the Appointment Cancellation Reason and Comments